in_toto/runlib.py

```diff
@@ -411,6 +410,7 @@ def in_toto_run(
     lstrip_paths=None,
     metadata_directory=None,
     use_dsse=False,
+    timeout=10
```

Comment:
Rather than a magic number here, let's retain the default value in settings.py and use it here?